indexer/osm_editor.cpp

```diff
@@ -556,6 +556,21 @@ bool Editor::GetEditedFeatureStreet(FeatureType const & feature, string & outFea
   return true;
 }
 
+vector<uint32_t> Editor::GetFeaturesByStatus(MwmSet::MwmId const & mwmId, FeatureStatus status) const
```

Comment:
Sorry, it's unclear from your message: the problem is that Editor depends on several 3party libraries or the problem is that its hard to use Editor class? Whatever, next time in pull requests related to editor + search be ready to provide unit tests, at least for basic functionality (create a test mwm with test features, modify some of them, check that search works).